portality/crosswalks/oaipmh.py

```diff
@@ -313,8 +315,8 @@ class OAI_DOAJ_Article(OAI_Crosswalk):
     def crosswalk(self, record):
         bibjson = record.bibjson()
 
-        metadata = etree.Element(self.PMH + "metadata", nsmap=self.NSMAP)
-        oai_doaj_article = etree.SubElement(metadata, self.OAI_DOAJ + "doajArticle")
+        metadata = etree.Element(self.PMH + "metadata")
+        oai_doaj_article = etree.SubElement(metadata, self.OAI_DOAJ + "doajArticle", nsmap=self.NSMAP)
         oai_doaj_article.set(self.XSI + "schemaLocation",
             "http://www.openarchives.org/OAI/2.0/ http://www.openarchives.org/OAI/2.0/OAI-PMH.xsd http://doaj.org/features/oai_doaj/1.0/ https://doaj.org/static/doaj/doajArticles.xsd")
```

Comment:
@richard-jones I am unsure about the schemaLocation that should be added here. Could you help?

Reply:
This looks correct to me, I think